src/Microsoft.Framework.DependencyInjection.Abstractions/project.json

```diff
@@ -8,7 +8,7 @@
         "net45": {
         },
         "dnx451": { },
```

Comment:
@ddobric, there are two more blog posts by Oren Novotny, that are worth to read (just to have this information spread a bit):
http://oren.codes/2015/06/09/pcls-net-core-dnx-and-uwp/
http://oren.codes/2015/07/29/targeting-net-core/
They clarified things quite well for me.